main/templates/main/tenants.html

```diff
@@ -32,7 +33,7 @@
 				<span class="trend" normal-min="{{ t.normal_min }}"> {{ t.trend }}</span>
 			</td>
 			<td>
-				<button type="button" class="btn btn-default receipts"><span class="glyphicon glyphicon-envelope"></span> {% trans "write letter"|title %}</button>
```

Comment:
I thought i good to have the property info also in the main panel, to have a good overview. Later, to be able to model reality, I think it would be a good idea to separate the tenant class into two classes: 1.  tenant (the person) and 2. tenancy (the duration of a rental)